src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -214,6 +224,42 @@ static ConfigDef newConfigDef()
         3,
         ConfigDef.Width.NONE,
         BUFFER_FLUSH_TIME_SEC)
+      .define(SNOWFLAKE_METADATA_ALL,
+        Type.STRING,
+        SNOWFLAKE_METADATA_DEFAULT,
+        Importance.LOW,
+        "Flag to control whether there is metadata collected",
```

Comment:
a little bit confuse.
You can add one more sentence to high it will disable all metadata when turning off.

Reply:
ack